config.yml

```diff
@@ -202,8 +240,6 @@ crafting:
     CRACKED_STONE_BRICK:
       material: SMOOTH_BRICK
       durability: 2
-    MOSSY_COBBLE:
-      material: MOSSY_COBBLESTONE

```

Comment:
Pretty sure this is a regression; might need to rebase against master and reissue your pull request.